packages/domain-picker/package.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "@automattic/domain-picker",
+	"version": "1.0.0-alpha.0",
+	"description": "Domain picker.",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/domain-picker"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"types": "dist/types",
+	"scripts": {
+		"clean": "check-npm-client && npx rimraf dist ../../.tsc-cache/packages__domain-picker*",
+		"prepare": "check-npm-client && tsc --project ./tsconfig.json && tsc --project ./tsconfig-cjs.json && copy-assets",
+		"prepublish": "check-npm-client && yarn run clean",
+		"watch": "check-npm-client && tsc --project ./tsconfig.json --watch"
+	},
+	"dependencies": {
+		"tslib": "^1.10.0"
+	},
+	"peerDependencies": {
```

Comment:
If you are not publishing the package, what is the reason to specify this as peer dependencies instead of regular dependencies?

Reply:
I didn't mean everything should be a peerDependency, that's not the case.

At the very least, `react` and `@wordpress/data` are problematic when when duplicated, `@wordpress/i18n` and `@wordpress/element` may also be. We've pushed them to peerDependencies in some other packages to avoid duplication.

If it's a private package we won't be not publishing, the distinction may be irrelevant, I'm not sure.

Reply:
> I didn't mean everything should be a peerDependency, that's not the case.

@sirreal Whoops! Sorry for that! I can't quite tell what should belong in `dependencies` and what should belong in `peerDependencies`. Dependencies that require others to have the same version belongs to `peerDependencies`?